### DISCUSSION TITLE
Crud service returns error ERR_STREAM_PREMATURE_CLOSE during massive export

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#567] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.5

### Description

Hi,
I have a crud-service `v6.10.0` that during a massive export call after 30s the stream download interrupts with a `ERR_STREAM_PREMATURE_CLOSE` error. The collection contains about 1M documents.

There is some solution to this error?

Thanks

### Actual Outcome

download stream interrupted

### Expected Outcome

be able to download the collection

**Comment by mattia-fumo** [comment]
Hi,
you could try to split the dataset into smaller chunks using `_l` (limit) and `_sk` (skip) query parameters of the `/export` endpoint

**Comment by fredmaggiowski** [comment]
Hi,

`ERR_STREAM_PREMATURE_CLOSE` seems like someone in the middle is closing the connection that is taking too much, what generally happens is the API Gateway being configured with a timeout, I'd check yours to see if that's the case as well.

**Comment by giovanni-motterle-mia-care** [comment]
Hi @elmalakomar, have you fixed this issue?
